action.yaml

```diff
@@ -40,15 +40,8 @@ runs:
       id: compose-args
       shell: bash
       run: |
-        encoded_github="$(echo ${GITHUB_CONTEXT} | base64 -w 0)"
```

Comment:
Please do note we have added base64 encoding due to an earlier bug that broke the json due to 

a commit message with quotes inside. https://github.com/philips-labs/slsa-provenance-action/issues/116

Could you confirm this isn't reintroducing that issue?

Reply:
Probably a `toJSON(github)` error? I'm going to check that. Thank you to point me out to this issue.